src/binder/bind/bind_ddl.cpp

```diff
@@ -195,11 +205,6 @@ BoundCreateTableInfo Binder::bindCreateRelTableInfo(const CreateTableInfo* info)
         clientContext->shouldUseInternalCatalogEntry());
 }
 
-static std::string getRelGroupTableName(const std::string& relGroupName,
-    const std::string& srcTableName, const std::string& dstTableName) {
-    return relGroupName + "_" + srcTableName + "_" + dstTableName;
-}
-
 BoundCreateTableInfo Binder::bindCreateRelTableGroupInfo(const CreateTableInfo* info) {
```

Comment:
We should throw exception when there is storage direction option specified. We are not handling it right now.